src/test/kotlin/com/zepben/evolve/services/common/ServiceDifferencesTest.kt

```diff
@@ -16,14 +16,14 @@ import org.hamcrest.Matchers.*
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.extension.RegisterExtension
 
-class ServiceDifferencesTest {
+internal class ServiceDifferencesTest {
 
     @JvmField
     @RegisterExtension
-    var systemOutRule: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
+    var systemOut: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
```

Comment:
todo: Should have been renamed to `systemErr`

Reply:
Renamed as such.

Reply:
Renamed as such.